### DISCUSSION TITLE
System.forcearm not starting the motor as IDLE for bolt

### DISCUSSION THREAD

**Discussion by maichongju** [bitcraze/discussions#229] [Q&A] [answered]
I followed the video tutorial for building the firmware for the bolt. Where it is recommended to set MOTORS_START_DISARMED to yes. I did that and set the idle thrust to 7000. However, when I connect the drone with the `cfclient` and set `1` for `system.forceArm`. The thrust indicator shows the thrust for each motor is 10% on idle, but the motor did not turn on.  

CONFIG_MOTORS_START_DISARMED=y
CONFIG_MOTORS_DEFAULT_IDLE_THRUST=7000

The motor I am using is RCINPOWER GTS V2 1202.5 6000KV MOTOR. Is it possible that the bolt is sending the single to the motor, but it just too low for the motor to operate?

**Comment by maichongju** [comment, marked as answer]
Well. I figure it out. If i change the IDLE RMP to higher value it works. So I guess recommended IDLE RMP does not work on every monitor.

**Reply by knmcguire** [reply]
Ah great to hear! thanks for letting us know
